src/kinetics/ReactionFactory.cpp

```diff
@@ -253,7 +253,7 @@ bool isThreeBody(const Reaction& R)
        if (trunc(reac.second) != reac.second) {
            return false;
        }
-       nreac += reac.second;
+       nreac += (size_t)reac.second;
```

Comment:
In this case, it may be cleaner to just change the type of `nreac` to `int`.

Reply:
Actually, this is due to `reac.second` being `double`.

Reply:
Hm. But I guess this is another case for `static_cast`? (which I just realized)